src/Microsoft.AspNetCore.Server.Kestrel.Transport.Libuv/Internal/Infrastructure/EventSourceExtensions.cs

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Server.Kestrel.Internal.Http;
+
+namespace Microsoft.AspNetCore.Server.Kestrel.Internal.Infrastructure
+{
+    public static class EventSourceExtensions
+    {
+        public static void ConnectionStart(this KestrelEventSource eventSource, Connection connection)
```

Comment:
This change was required to decouple `Connection` which is in Transport.Libuv and `KestrelEventSource` which is now in Core. ConnectionStop/Start are still on KestrelEventSource which is in Kestrel.Core, but the public methods now take string arguments directly instead of `Connection`.